db/schema.rb

```diff
@@ -11,6 +11,8 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema[7.1].define(version: 2024_08_07_224704) do
+  create_schema "analytics"
+
```

Comment:
huh, i don't understand where this came from?

Reply:
When I merged main it came with that. I can re-remove it.

Reply:
hm yeah if it's on main it shouldn't show up as a change so i'd just make sure your schema.rb matches main since you didn't do any db stuff as part of this PR